### ISSUE TITLE
Improve Error/Exception Handling

### ISSUE THREAD

**Issue by FlorianDecker** [rubicon-oss/LicenseHeaderManager#17]
When an Exception gets thrown, the user should be possible to see the type of the exception and the stacktrace for better error reporting.

**Comment by drauch** [comment]
Probably report the exception to the LHM VS output window?

**Comment by MichaelKetting** [comment]
Good idea. I think we created the feature request before figuring out how to do the VS outpout window.
